modules/mobile/app/controllers/mobile/v0/claims_and_appeals_controller.rb

```diff
@@ -138,6 +138,9 @@ def get_response_status(errors)
       def filter_by_date(start_date, end_date, list)
         list.filter do |entry|
           updated_at = entry[:updated_at]
+
+          next(true) unless updated_at
```

Comment:
The bug appears to be for the date_filed in the adapter. Is this just an abundance of caution? 

Also, is the `next(true)` to ensure that it returns true so it's not filtered out?

Reply:
Both dates are non-nullable according to the LH docs but if one is found null in test data, I figure the other could easily be. 

for the `next(true)`, yes, that allows the nil date claims to still be included in the list without being filtered out for not being within the date field. historically, we always err on the side of caution and include entries even if it's not fully filled out.